src/Pages/Withdrawals.razor

```diff
@@ -476,21 +477,22 @@
 
         _isFinanceManager = ClaimsPrincipal != null && ClaimsPrincipal.IsInRole(ApplicationUserRole.FinanceManager.ToString());
         await GetData();
+        await LoadColumnLayout();
```

Comment:
For some reason this page refuses to behave like the rest. So I'm forced to load the column layout twice